### ISSUE TITLE
torch.dropout与 paddle.nn.functional.dropout的映射关系有问题

### ISSUE THREAD

**Issue by Zhenkun-Zhang** [PaddlePaddle/docs#7328]
### 文档链接&描述 Document Links & Description

docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.dropout.md

torch.dropout：内建函数（是 TorchScript 在编译阶段可直接使用的低级 API，通常不是在 torch.* Python 层显式调用的）不是公开 Python API，而是一个内置的 C++ 算子。torch.dropout(input: Tensor, p: float, train: bool) -> Tensor

paddle.nn.functional.dropout：普通函数
https://github.com/PaddlePaddle/docs/blob/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.dropout.md
torch.nn.functional.dropout：普通函数

### 请提出你的建议/提个PR修复这个问题 Please give your suggestion/update a PR to fix it

建议直接删掉docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.dropout.md

**Comment by Zhenkun-Zhang** [comment]
https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.dropout.html

**Comment by JiwaniZakir** [comment]
Working on it.

**Comment by zhwesky2010** [comment]
这个映射是直接对应两种的用法，并不是一定需要是Python还是C++，只要有这种`torch.**`用法即可映射。
